docs/cookbooks/templates/theme_generated_elements.rst

```diff
@@ -0,0 +1,125 @@
+Elements generated on theme installation
+========================================
+
+To provide better experience for end users after theme installation Publisher can create some default content used by theme.
+Thanks to this feature theme developer can be sure that all of his article or category templates will be visible without
+complicated configuration by end user.
+
+Generated elements can be declared in theme.json config file under :code:`generatedData` key. Example:
+
+.. code-block:: json
+
+    {
+     "name": "my/custom-theme",
+     "generatedData": { ... }
```

Comment:
I would rename `generatedData` to `defaultData` or `demoData`

Reply:
I had here before default data, but i changed it to generatedData as all of those elements are generated (articles are even using faker library). I want to have it clear that it will be created on theme installation.